### PR TITLE
ENG-8309 Increment packet number

### PR DESCRIPTION
**Pull request by jeff-niemann**

### CODE REVIEW

File: NeuroID/NeuroIDClass/Extensions/NIDPacketNumber.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import Foundation
+
+public extension NeuroID {
+    static func getPacketNumber() -> Int32 {
+        return packetNumber
+    }
+    
+    static func incrementPacketNumber () {
+        packetNumber += 1
+    }
```
Comment (kevinsites) [inline NeuroID/NeuroIDClass/Extensions/NIDPacketNumber.swift:17]: Are we forever incrementing? Does this number need to be reset to 0 on a new session or?

**Comment by kishisaka-nid** [comment]
so this resets to zero everytime the app/SDK is restarted? I don't see this being stored anywhere. should this be restated when the user calls startAppFlow or start or startSession?

File: NeuroID/NeuroIDClass/Extensions/NIDPacketNumber.swift (hunk 1 above)
Comment (jeff-niemann) [reply NeuroID/NeuroIDClass/Extensions/NIDPacketNumber.swift:17]: Oooh good call!! I’ll make a reset

Comment (jeff-niemann) [reply NeuroID/NeuroIDClass/Extensions/NIDPacketNumber.swift:17]: Added reset when configure is called... this should closely match what happens in web

**Review by kevinsites — DISMISSED** [review]

**Review by kevinsites — APPROVED** [review]

**Review by kishisaka-nid — APPROVED** [review]

**Pull request merged**